postreise/plot/plot_bar_generation_max_min_actual.py

```diff
@@ -0,0 +1,97 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+from powersimdata.network.model import ModelImmutables
+
+from postreise.analyze.generation.summarize import (
+    sum_generation_by_state,
+    sum_generation_by_type_zone,
+)
+
+
+def plot_bar_generation_max_min_actual(
+    scenario,
+    interconnect,
+    gen_type,
+    percentage=False,
+    show_as_state=True,
+    fontsize=15,
+    plot_show=True,
+):
+    """Generate for a given scenario the bar plot of maximum generation vs. actual
```

Comment:
Should we use total capacity in place of maximum generation?

Reply:
Done.